addons/account/models/account_move.py

```diff
@@ -4817,7 +4841,7 @@ def default_get(self, default_fields):
             if 'account_id' in default_fields and not values.get('account_id'):
                 if len(move.line_ids[-2:]) == 2 and  move.line_ids[-1].account_id == move.line_ids[-2].account_id != False:
                     values['account_id'] = move.line_ids[-2:].mapped('account_id').id
-        if values.get('display_type') or self.display_type:
+        if values.get('line_type') in ['invl_note', 'invl_section'] or self and self.is_display_line():
```

Comment:
redundant `or self`

Reply:
I don't think so, if self is an empty recordset, calling is_display_line() will raise an error